sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v1/init.sql

```diff
@@ -83,6 +94,7 @@ CLUSTER BY
       adjust_adgroup,
       adjust_campaign,
       adjust_creative,
+      reported_first_session_ping AS is_new_install,
```

Comment:
Does this column means that the client was a new install on this submission_date?
Cause the `reported_first_session_ping` means that client ever reported a first_session ping but not necessarily on this date so that'd may be different concepts.

Reply:
Good point. Update to just count a new install on their first day.